arbor/fvm_layout.cpp

```diff
@@ -433,6 +436,12 @@ fvm_cv_discretization fvm_cv_discretize(const cable_cell& cell, const cable_cell
             D.init_membrane_potential[i] /= D.cv_area[i];
             D.temperature_K[i] /= D.cv_area[i];
         }
+        else if (p!=-1) {
```

Comment:
This will only turn up if there is a trivial CV at the root with multiple top level branches. There isn't really a consistent choice of how to pick a starting membrane potential or temperature here, though some heuristic could be picked. Pick the first branch values, perhaps?